addons/web_editor/static/src/components/media_dialog/media_dialog.js

```diff
@@ -245,6 +245,11 @@ export class MediaDialog extends Component {
             element.classList.remove('o_modified_image_to_save');
             element.classList.remove('oe_edited_link');
             element.classList.add(...TABS[this.state.activeTab].Component.mediaSpecificClasses);
+            if (this.props.node 
+                && this.state.activeTab == "IMAGES"
+                && (this.props.node.classList.contains("fa") || this.props.node.classList.contains("to_icon"))) {
+                element.classList.add("to_icon");
+            }
```

Comment:
Hello 👋 I'm sorry to nitpick but I had to read this twice to realize that there was in fact something going on inside the `if`. I suggest the following indentation instead which is rather standard and is more readable IMO:
```suggestion
if (
    this.props.node &&
    this.state.activeTab == "IMAGES" &&
    (this.props.node.classList.contains("fa") || this.props.node.classList.contains("to_icon"))
) {
    element.classList.add("to_icon");
}
```
I suppose the last condition might be split too depending on line length.

Reply:
Hello ! No problem with nitpicking, I'm still learning :D
I'm sorry for the email, I made some mistake with draft/open PR (as I said, learning :p).